RecoTracker/LSTCore/interface/Common.h

```diff
@@ -35,6 +35,13 @@ namespace lst {
 
   constexpr unsigned int size_superbins = 45000;
 
+  constexpr uint16_t kTCEmptyLowerModule = 0xFFFF;     // Sentinel for empty lowerModule index
+  constexpr unsigned int kTCEmptyHitIdx = 0xFFFFFFFF;  // Sentinel for empty hit slots
+  constexpr int kPackedScoreShift = 32;
+  constexpr int kPackedIndexShift = 4;
+  constexpr unsigned int kPackedIndexMask = 0xFFFFFFF;
+  constexpr unsigned int kPackedSlotMask = 0xF;
```

Comment:
these are used only in `struct ExtendTrackCandidatesFromDupT5`
Perhaps move them there, no need to see these in the "global" `lst` namespace